packages/@aws-cdk/aws-servicecatalogappregistry/lib/target-application.ts

```diff
@@ -90,10 +90,14 @@ class CreateTargetApplication extends TargetApplication {
   }
   public bind(scope: Construct): BindTargetApplicationResult {
     const stackId = this.applicationOptions.stackId ?? 'ApplicationAssociatorStack';
+    (this.applicationOptions.description as string) =
+            this.applicationOptions.description || 'Stack containing AppRegistry Application';
+    (this.applicationOptions.env as cdk.Environment) =
+            this.applicationOptions.env || { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION };
     const applicationStack = new cdk.Stack(scope, stackId, this.applicationOptions);
     const appRegApplication = new Application(applicationStack, 'DefaultCdkApplication', {
       applicationName: this.applicationOptions.applicationName,
-      description: this.applicationOptions.applicationDescription,
+      description: this.applicationOptions.applicationDescription || 'AppRegistry Application to which all stacks inside this cdkApp will get associated.',
```

Comment:
When you look at the description in the console, the words "this cdkApp" are meaningless.

Reply:
We are trying to tell the purpose of the creation of the app. I agree `this cdkApp` is useless. How about this ? 
AppRegistry Application to which all stacks inside the cdkApp will get associated.